Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/cart/Cart.kt

```diff
@@ -150,12 +158,82 @@ private fun CartContent(
             )
         }
         items(orderLines) { orderLine ->
-            CartItem(
-                orderLine = orderLine,
-                removeSnack = removeSnack,
-                increaseItemCount = increaseItemCount,
-                decreaseItemCount = decreaseItemCount,
-                onSnackClick = onSnackClick
+            SwipeDismissItem(
+                background = { offsetX ->
+                    // Background color changes from light gray to red when the swipe to delete with exceeds 160.dp
+                    val backgroundColor = if (offsetX < -160.dp) {
+                        JetsnackTheme.colors.error
+                    } else {
+                        JetsnackTheme.colors.uiFloated
+                    }
+                    Column(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .fillMaxHeight()
+                            .background(backgroundColor),
+                        horizontalAlignment = Alignment.End,
+                        verticalArrangement = Arrangement.Center
+                    ) {
+                        // Set 4.dp padding only if offset is bigger than 160.dp
+                        val padding = if (offsetX > -160.dp) 4.dp else 0.dp
```

Comment:
Does this cause a jump? Wrap in `animateDpAsState`?

Reply:
Done